reference/slot.md

```diff
@@ -13,15 +13,15 @@ Slot {
 added in the `endPoint` field of a [Slot](#slot) before to be able to add any
 node in the `endPoint` field of the second [Slot](#slot). Furthermore, the
 second [Slot](#slot) can be added only if it has the same `type` as the first
-one.  The [Slot](#slot) node is particularly usefull with PROTOs, it allows the
+one. The [Slot](#slot) node is particularly useful with PROTO nodes, it allows the
 user to constrain the type of nodes that can be added in an extension field of
 the PROTO. Imagine for example that you have an armed robot in which you can
 plug different kinds of hands. In order to do so you will put the hand as an
 extension field of your robot, you will then be able to add all the different
-PROTOs of hand that you have made. But nothing prevent you to add a PROTO of
+PROTO models of hand that you have made. But nothing prevents you from adding a PROTO of
```

Comment:
I don't know.
I would say yes for consistency, but we are not doing it for node names ('Solid', 'LightSensor', etc.).